pkg/components/storage.go

```diff
@@ -71,6 +71,12 @@ func startCSIPlugin(cfg *config.MicroshiftConfig, kubeconfigPath string) error {
 		}
 	)
 
+	// check for lvm utility 'vg', if not present, log and do not start CSI plugin
+	if ! lvmd.LvmSupported() {
+		klog.Warning("LVM utility 'vg' not found in $PATH, skipping CSI deployment")
```

Comment:
This warning message assumes knowledge here about how the utility function in another package works. You could avoid leaking that knowledge by having LvmSupported() return the error and logging that directly.